ultravox/tools/ds_tool/ds_tool.py

```diff
@@ -47,6 +47,7 @@ def map_split(
         num_proc: int,
         writer_batch_size: int,
         exclude_fields: List[str],
+        include_filter: bool = False,
```

Comment:
Adding this flag here because for some datasets we don't expect there to be any empty text values, and for those we don't want to run through an unncessary filter map since its pretty time consuming.

Reply:
Might also be able to just count the number of template failures and only filter when != 0.